### PR TITLE
Updated assembly directory to specific hifi2genome and added another hifi dataset

### PR DESCRIPTION
**Pull request by elizabethmcd**
This pull requests modifies the assembly directory and renames it to hifi2genome to reflect a new workflow scope that will be specific to just assembling and profiling PacBio Hifi reads into single genomes since for most higher order eukaryote sequencing this is the avenue we will be taking. This also adds an additional HiFi dataset of recently failed Colpoda sequencing with very low yield where the resulting fastq was only 5Mb which I subsetted to 100 reads and gzipped. This addresses #6

### CODE REVIEW

**Comment by elizabethmcd** [comment]
Updated - removed the Csteinii subset reads because I have a bad feeling they are trash. From some prelminary sourmash analysis of comparing to reference Colpoda shotgun sequencing there's a very very small amount similar by comparing sourmash signatures, but that might be from the very small amount of sequencing yield. Additionally from the seqqc results so far there's a little bit of contamination, so I'm not quite confident in these reads yet to even use as a simple test case.

File: hifi2genome/samplesheet_test.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+sample,fastq_1,fastq_2
+csteinii,https://github.com/Arcadia-Science/test-datasets/raw/main/hifi2genome/csteinii_failed_hifi_100_subsampled.css.fq.gz,
```
Comment (taylorreiter) [inline hifi2genome/samplesheet_test.csv]: should this be removed here then?

**Review by taylorreiter — APPROVED** [review]
I think you still need to remove the line for the csteinii data in the csv file, but otherwise LGTM

File: hifi2genome/samplesheet_test.csv (hunk 1 above)
Comment (elizabethmcd) [reply hifi2genome/samplesheet_test.csv]: ah yes, just removed this

**Pull request merged**